.github/workflows/cicd.yml

```diff
@@ -151,6 +151,7 @@ jobs:
     uses: ./.github/workflows/smoke-tests.yml
     with:
       py_ver: ${{ needs.process-gitref.outputs.py_ver }}
+    secrets: inherit
```

Comment:
yeah, still no dice.

the shit is cursed.

Reply:
yeah, the challenge was with secrets not being picked up since the PR was opened from a fork.

Now after the merge, it is passing to. Thanks @hyposcaler-bot !